eng/versioning/version_client.txt

```diff
@@ -33,6 +33,8 @@ com.azure:azure-identity-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-eventhubs;5.1.2;5.2.0-beta.3
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.1.2;1.2.0-beta.3
 com.azure:azure-messaging-servicebus;7.0.0-beta.5;7.0.0-beta.6
+com.azure:azure-messaging-servicebus-track1-perf;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
Should the track 1 performance test use the `groupId` `com.microsoft.azure`? That way we don't need to include the statements of Track 1 and Track 2 in the artifact as that isn't exactly how we publicized the unified SDKs and the previous SDKs.